### PR TITLE
Add Row component 

### PR DESCRIPTION
**Pull request by Publicker**
This pull request introduces the `Row` component, intended to be used by the Table component in upcoming implementations. A Storybook story has been added for testing and visualisation purposes.

Changes include:

1. Creation of the Row component, designed to function as a <tr> HTML element within a table structure.
2. Inclusion of a temporary table as a Storybook decorator for enhanced display, to be replaced with the proper Table component in the future.

It should be noted that the `Status` and `Ellipsis` components have not been included in this PR, as they are currently under development separately.

--- 

Added the following stories with corresponding links:

- [Row / Example Playground](https://6410c09a16b1ae3112580b1c-fycgrmzdwz.chromatic.com/?path=/story/row--playground)
- [Row / Example Today](https://6410c09a16b1ae3112580b1c-fycgrmzdwz.chromatic.com/?path=/story/row--example-today)
- [Row / Example Yesterday](https://6410c09a16b1ae3112580b1c-fycgrmzdwz.chromatic.com/?path=/story/row--example-yesterday)
- [Row / Example Two Days Ago](https://6410c09a16b1ae3112580b1c-fycgrmzdwz.chromatic.com/?path=/story/row--example-two-days-ago)
- [Row / Example Two Years Ago](https://6410c09a16b1ae3112580b1c-fycgrmzdwz.chromatic.com/?path=/story/row--example-two-years-ago)
- [Row / Example GBP](https://6410c09a16b1ae3112580b1c-fycgrmzdwz.chromatic.com/?path=/story/row--example-gbp)

### CODE REVIEW

**Review by donalnofrixion — APPROVED** [review]

**Review by arifmatin — APPROVED** [review]

File: src/components/Row/Row.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+interface RowProps {
+  status: 'paid' | 'partial' | 'unpaid';
+  createdAt: Date;
+  contact: LocalContact;
+  amount: number;
+  currency: 'EUR' | 'GBP';
+  tags: string[];
+}
+
+const Row = ({ status, createdAt, contact, amount, currency, tags }: RowProps) => {
```
Comment (donalnofrixion) [inline src/components/Row/Row.tsx:14]: Just thinking. Would it not make more sense to call this PaymentRequestRow and pass a PaymentRequest into it? In future the PaymentRequest Details will be triggered here? I don't think it makes sense to create a resuable row component at this level. Other views will probably have completely different fields/details.

Comment (donalnofrixion) [reply src/components/Row/Row.tsx]: Fair enough. We should probably look at trimming down our ApiRsesponses as well. I created them as dupes of the api versions for starters.

Comment (Publicker) [reply src/components/Row/Row.tsx]: Yep. That's why I started creating "Local" types (i.e `LocalCurrency`), following the advice from our [bible](https://martinfowler.com/articles/modularizing-react-apps.html).

In a near future we'd find ourselves mapping the API types to the Local ones.

---

Btw, the renaming is already up🤘🏻

**Review by pabloi-nf — CHANGES_REQUESTED** [review]
image

Comparing the Figma file with the HTML I see some small things to fix:

- Font size for date and customer name should be 13px.
- Distance between customer name and customer email should be smaller.
- Currency text should be 14px.
- Apply Inter font variants for characters 1, 4,  and 9.

**Comment by Publicker** [comment]
> image
> 
> Comparing the Figma file with the HTML I see some small things to fix:
> 
> * Font size for date and customer name should be 13px.
> * Distance between customer name and customer email should be smaller.
> * Currency text should be 14px.
> * Apply Inter font variants for characters 1, 4,  and 9.

@pabloi-nf All these items were attended, please take a look at the new version [here](https://6410c09a16b1ae3112580b1c-hzqjaqchlu.chromatic.com/?path=/story/payment-request-row--playground).

**Comment by Publicker** [comment]
@pabloi-nf Hover state & onClick prop added too!

Check it in [here](https://6410c09a16b1ae3112580b1c-rdmzvixuof.chromatic.com/?path=/story/payment-request-row--playground)!

**Review by pabloi-nf — APPROVED** [review]

**Pull request merged**